### ISSUE TITLE
NVDA announcing pagination poorly

### ISSUE THREAD

**Issue by BlueSlug** [inclusive-design/wecount.inclusivedesign.ca#801]
## Describe the issue

In NVDA (and potentially other screen readers), the pagination links on the Resources, Initiatives and News pages are announced as simply "1", "2" ... "45", "next". These should instead be announced as "Page 1" (and so on) and "next page"

## To Reproduce

1. Go to the Resources page using NVDA (or, if affected, another screen reader)
2. Navigate to the pagination links
3. Experience the issue

## Expected behavior

The page links should be announced with sufficient context to understand their purpose and destination.

## Screenshots

N/A

## Applicable WCAG Success Criteria:

- [1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Desktop:

- OS: Windows 10 Pro 21H1
- Browser: Chrome 96
- Assistive Technologies: NVDA 2019.3.1

## Smartphone:

N/A

## Additional context

NVDA does indicate that it is in the "pagination" landmark, which helps, so this aspect should be explored before beginning work on this issue.
